src/index.js

```diff
@@ -345,6 +346,7 @@ Strings.DEFAULTS = {
   [Strings.QUOTA]: 'Quota',
   [Strings.RULE]: 'Rule',
   [Strings.SALE]: 'Sale',
+  [Strings.TRANSACTION_DATE]: 'Transaction Date',
```

Comment:
Think we should do anything special for the default abbreviation of this, or do you think `'TD'`/`'td'` is fine?

Was wondering if something like `'TrxDt'` or `'trx'` would be better. Ideally would use a pattern that works for "Effective Date"  too (e.g. `'EffDt'` or `'eff'`), but it's probably not worth the effort of coding a special case here. Just interested in others' thoughts.

Reply:
Ignoring this for now.